conjure-java-core/src/main/java/com/palantir/conjure/java/types/ErrorGenerator.java

```diff
@@ -165,24 +166,25 @@ private JavaFile generateErrorTypesForNamespace(
                                                     .build()))
                                     .map(arg -> {
                                         TypeName argumentTypeName = typeMapper.getClassName(arg.getType());
-                                        Optional<LogSafety> required = arg.getSafety();
+                                        Optional<LogSafety> underlyingTypeSafety = SafetyUtils.getUsageTimeSafety(arg);
```

Comment:
renamed because the name "required" didn't seem to fit anymore and was confusing me